doc/040_backup.rst

```diff
@@ -146,6 +146,15 @@ change detection rule based on file metadata to determine whether a file is
 likely unchanged since a previous backup. If it is, the file is not scanned
 again.
 
+The previous backup, called parent snaphot in restic terminology, is
+determined as follows. By default restic selects the last backup using the
+same hostname and backup paths. You can change the selection criteria using
```

Comment:
It also uses the exclude list, right?

Reply:
No. The exclude list isn't used anywhere for grouping snapshots.